src/executors/default.yml

```diff
@@ -26,7 +26,7 @@ docker:
     environment:
       POSTGRES_PASSWORD: postgres
   - image: redis:<< parameters.redis >>
-  - image: docker.elastic.co/elasticsearch/elasticsearch:<< parameters.elasticsearch >>
+  - image: elastic/elasticsearch:<< parameters.elasticsearch >>
```

Comment:
Does this need the `elastic/` prefix? Looks like `elasticsearch` is the official image similar to `redis` and `postgres`

Reply:
https://hub.docker.com/_/elasticsearch